.github/workflows/publish-dotnet.yaml

```diff
@@ -0,0 +1,44 @@
+name: Publish on Release
+
+on:
+  release:
+    types: [published]
+
+defaults:
+  run:
+    shell: bash
+    working-directory: dotnet
+
+jobs:
+
+  publish:
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          source-url: https://api.nuget.org/v3/index.json
+          dotnet-version: 5.0.x
+        env:
+          NUGET_AUTH_TOKEN: ${{ secrets.NUGET_TOKEN }}
+      - name: Normalize version string
+        id: normalize_version
+        # Remove leading 'v' from version string
+        run: |
+          version="${{ github.event.release.tag_name }}"
+          echo "::set-output name=VERSION::`echo ${version:1}`"
+      - name: Find and Replace Version templates
+        run: |
+          sed -i.bak 's/${VERSION}/${{ steps.normalize_version.outputs.VERSION }}/g' Nitric.Sdk/Nitric.Sdk.csproj
+          rm Nitric.Sdk/Nitric.Sdk.csproj.bak
```

Comment:
Are these right?

Reply:
Nope, thanks - I'll fix that today

Reply:
Fixed with updated name and path relative to the action's working directory